### ISSUE TITLE
training options in experiments do not match with paper

### ISSUE THREAD

**Issue by bsvineethiitg** [xingyizhou/CenterNet#96]
In the paper you mention that the offset regression and the bbox size ("wh") regression both are of size H/R X W/R X 2. That is, you regress 2 values each per pixel in a dense way.

But in your training arguments under experiments folder do not mention dense_wh option. Moreover, you do not seem to use dense version of offset regression in contrast to what you mentioned in the paper.

Can you please explain the difference between the dense version versus the version in the code. Can you confirm if the training commands provided under experiments are the one used in the paper? 

Thanks

**Comment by xingyizhou** [comment]
Thanks for the challenge. `--dense_wh` is never used in the experiments. By default, the loss is only applied in the positive locations of the size map, other positions are ignored (set to 0 weight). This is implemented by the `transpose_and_gather_feat` function. `--dense_wh` apply loss not only to the positive locations, but also to the pixels near the positive location in a Gaussian weight. But we didn't find it helpful. The paper describes exactly the default version.

**Comment by bsvineethiitg** [comment]
Awesome! That clears up. So, is it the same case with Regression Loss as well for the offsets? Where you only include the positive grid cell points in the loss function?

**Comment by xingyizhou** [comment]
Yes.
